src/CBLReplicatorConfig.hh

```diff
@@ -225,6 +231,18 @@ namespace cbl_internal {
                 }
                 enc.endDict();
             }
+            
+            enc.writeKey(slice(kC4ReplicatorOptionMaxRetries));
```

Comment:
LiteCore has its own defaults for these. It would be cleaner to use those rather than always override all of the properties. If LiteCore's defaults don't match CBLs, then we should change LiteCore's.